coralproject/talk
# [CORL-3202] don't show the `Refresh comments` button when live updates are disabled

## What does this PR do?

- turns off the `Refresh comments` button when live updates are disabled in the tenant settings

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

N/A

## How do I test this PR?

- disable live updates under `Admin > Configure > Advanced > Comment stream live updates`
- go to stream
- leave stream tab and come back
- no `Reload comments` button/prompt should show

## Were any tests migrated to React Testing Library?

No

## How do we deploy this PR?

- Merge into `develop`
